### PR TITLE
Feature/version update 2

### PR DESCRIPTION
**Pull request by kerriannercrawford**
- Added functionality for splitResult
- Added functionality for JSONata transformation
- Added functionality for childArray
- Changed references of `body` to `data`

Vulnerabilities still exist. Ran `npm audit fix` and fixed the ones that did not create breaking changes, but when running `npm audit fix --force`, introduced breaking changes. 

No unit tests for added functionality yet but tested component on local environment and everything seems to work as expected.

### CODE REVIEW

File: spec/json2xml.spec.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ const inputMessage = {
 const expectedOutputStringWithHeaders = '<?xml version="1.0" encoding="UTF-8" standalone="no"?>\n<ORDERRESPONSE xmlns:ns2="http://www.bmecat.org/bmecat/2005" version="2.1">\n  <ORDERRESPONSE_HEADER>\n    <ORDERRESPONSE_INFO>\n      <ORDERRESPONSE_DATE>2020-04-07T09:07:45.188Z</ORDERRESPONSE_DATE>\n      <ORDER_ID>1234</ORDER_ID>\n    </ORDERRESPONSE_INFO>\n  </ORDERRESPONSE_HEADER>\n  <ORDERRESPONSE_ITEM_LIST/>\n</ORDERRESPONSE>';
 const expectedOutputStringWithoutHeaders = '<ORDERRESPONSE xmlns:ns2="http://www.bmecat.org/bmecat/2005" version="2.1">\n  <ORDERRESPONSE_HEADER>\n    <ORDERRESPONSE_INFO>\n      <ORDERRESPONSE_DATE>2020-04-07T09:07:45.188Z</ORDERRESPONSE_DATE>\n      <ORDER_ID>1234</ORDER_ID>\n    </ORDERRESPONSE_INFO>\n  </ORDERRESPONSE_HEADER>\n  <ORDERRESPONSE_ITEM_LIST/>\n</ORDERRESPONSE>';
 
-describe('JSON to XML', () => {
+xdescribe('JSON to XML', () => {
```
Comment (winklerj) [inline spec/json2xml.spec.js:41]: Is this a typo or intentional?

Comment (winklerj) [reply spec/json2xml.spec.js:41]: I see it on several other lines and that it will disable the test suite

**Review by winklerj — CHANGES_REQUESTED** [review]

File: lib/actions/attachmentToJson.js [hunk 2]
```diff
@@ -49,6 +49,8 @@ module.exports.process = async function processAction(msg, cfg) {
 
         this.logger.debug(`For provided filename response status: ${response.status}`);
 
+        this.logger.info('the response is ', response);
```
[the remark this answers is not in the document]
Comment (kerriannercrawford) [reply lib/actions/attachmentToJson.js:52]: Changed to debug. I added this for testing while I was working on the component. Used to searching for console.logs prior to committing, need to get into the habit of looking for this.logger.

File: spec/json2xml.spec.js (hunk 1 above)
Comment (kerriannercrawford) [reply spec/json2xml.spec.js:41]: The xdescribe was intentional, but not meant to be committed. I like to turn off the tests I'm not currently working on so I don't have to scroll through the results. Re-enabled all tests. (And taking notes so you don't have to keep finding the same silly things)

**Review by winklerj — APPROVED** [review]

**Pull request merged**